### PR TITLE
Update de_DE.lang (march 2016)

### PR DESCRIPTION
**Pull request by ghost**
Minor changes to localisations, feel free to disagree.

### CODE REVIEW

File: src/main/resources/assets/bloodmagic/lang/de_DE.lang [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #Creative Tab
-itemGroup.BloodMagic.creativeTab=Blood Magic
-itemGroup.BloodMagic.creativeTabTome=Blood Magic Upgradeschwarte
+itemGroup.BloodMagic.creativeTab=Blutmagie
```
Comment (Vexatos) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang:2]: Never translate a mod's name.

File: src/main/resources/assets/bloodmagic/lang/de_DE.lang [hunk 2]
```diff
@@ -9,9 +9,9 @@ item.BloodMagic.activationCrystal.creative.name=Kreativer Aktivierungskristall
 
 item.BloodMagic.sacrificialDagger.normal.name=Opfermesser
 item.BloodMagic.sacrificialDagger.creative.name=Kreatives Opfermesser
-item.BloodMagic.pack.selfSacrifice.name=Rucksack des Blutlassers
-item.BloodMagic.pack.sacrifice.name=Wappen
-item.BloodMagic.daggerOfSacrifice.name=Dolch der Opferung
+item.BloodMagic.pack.selfSacrifice.name=Weste des Blutlassens
+item.BloodMagic.pack.sacrifice.name=Vampirische Weste
+item.BloodMagic.daggerOfSacrifice.name=Schwert der Opferung
```
Comment (Vexatos) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang:14]: "dagger" means "Dolch", I think it should stay that way.

Maybe "Opferdolch"? "Opferungsdolch"?

File: src/main/resources/assets/bloodmagic/lang/de_DE.lang [hunk 3]
```diff
@@ -20,7 +20,7 @@ item.BloodMagic.bound.pickaxe.name=Gebundene Spitzhacke
 item.BloodMagic.bound.axe.name=Gebundene Axt
 item.BloodMagic.bound.shovel.name=Gebundene Schaufel
 
-item.BloodMagic.bucket.lifeEssence.name=Lebenseimer
+item.BloodMagic.bucket.lifeEssence.name=Eimer Lebensessenz
```
Comment (Vexatos) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang:23]: How about "Eimer mit Lebensessenz"? Would fit other mods' buckets a little better.

File: src/main/resources/assets/bloodmagic/lang/de_DE.lang [hunk 4]
```diff
@@ -64,13 +64,13 @@ item.BloodMagic.reagent.reductus.name=Reductus
 item.BloodMagic.reagent.potentia.name=Potentia
 
 item.BloodMagic.bloodShard.weak.name=Schwache Blutscherbe
-item.BloodMagic.bloodShard.demon.name=Dämonenblutscherbe
+item.BloodMagic.bloodShard.demon.name=Dämonische Blutscherbe
```
Comment (Vexatos) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang:67]: Not sure I like this one...

File: src/main/resources/assets/bloodmagic/lang/de_DE.lang [hunk 5]
```diff
@@ -64,13 +64,13 @@ item.BloodMagic.reagent.reductus.name=Reductus
...
 
 item.BloodMagic.baseComponent.reagentWater.name=Wasserreagenz
 item.BloodMagic.baseComponent.reagentLava.name=Lavareagenz
 item.BloodMagic.baseComponent.reagentAir.name=Luftreagenz
-item.BloodMagic.baseComponent.reagentFastMiner.name=Abbaureagenz
-item.BloodMagic.baseComponent.reagentVoid.name=Leerenreagenz
+item.BloodMagic.baseComponent.reagentFastMiner.name=Reagenz der Eeile
```
Comment (Vexatos) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang:72]: "Eile"

File: src/main/resources/assets/bloodmagic/lang/de_DE.lang [hunk 6]
```diff
@@ -64,13 +64,13 @@ item.BloodMagic.reagent.reductus.name=Reductus
...
+item.BloodMagic.baseComponent.reagentVoid.name=reagenz der Leere
```
Comment (Vexatos) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang:73]: Mind your capitalization.

File: src/main/resources/assets/bloodmagic/lang/de_DE.lang [hunk 7]
```diff
@@ -85,13 +85,13 @@ item.BloodMagic.sigil.compression.name=Siegel der Kompression
 item.BloodMagic.sigil.divination.name=Wahrheitssiegel
 item.BloodMagic.sigil.water.name=Wassersiegel
 item.BloodMagic.sigil.lava.name=Lavasiegel
-item.BloodMagic.sigil.void.name=Leerensiegel
+item.BloodMagic.sigil.void.name=Siegel der Leere
 item.BloodMagic.sigil.greenGrove.name=Siegel des Grünen Hains
 item.BloodMagic.sigil.elementalAffinity.name=Siegel der elementaren Verbundenheit
-item.BloodMagic.sigil.haste.name=Siegel der Hast
+item.BloodMagic.sigil.haste.name=Siegel der Geschwindigkeit
```
Comment (Vexatos) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang:91]: I like the word "Hast", actually; it flows much more nicely off the tongue.

File: src/main/resources/assets/bloodmagic/lang/de_DE.lang [hunk 8]
```diff
@@ -85,13 +85,13 @@ item.BloodMagic.sigil.compression.name=Siegel der Kompression
...
 item.BloodMagic.sigil.suppression.name=Siegel der Unterdrückung
 item.BloodMagic.sigil.magnetism.name=Siegel des Magnetismus
-item.BloodMagic.sigil.fastMiner.name=Siegel des Schnellen Abbaus
+item.BloodMagic.sigil.fastMiner.name=Siegel des Eile
```
Comment (Vexatos) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang:94]: :+1:

File: src/main/resources/assets/bloodmagic/lang/de_DE.lang [hunk 9]
```diff
@@ -201,12 +201,12 @@ tooltip.BloodMagic.bound.axe.desc=&oDämonischer Kahlschlag
 tooltip.BloodMagic.bound.shovel.desc=&oSchrubbt den Dreck vom Boden
 
 tooltip.BloodMagic.sacrificialDagger.desc=Schon ein Stich in den Finger genügt
-tooltip.BloodMagic.slate.desc=Infundierter Stein in einem Blutaltar
+tooltip.BloodMagic.slate.desc=Stein getränkt im Blutaltar
```
Comment (Vexatos) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang:204]: This is a good one. :+1:

File: src/main/resources/assets/bloodmagic/lang/de_DE.lang (hunk 2 above)
Comment (ghost) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang:14]: If we call the sacrificial knife "Opfermesser" then we should call the sacrificial dagger "Opferdolch"

Comment (Vexatos) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang:14]: Sounds good, I think.

**Comment by WayofTime** [comment]
If there are any more changes, please make a new PR.

**Pull request merged**